docs/user-guide/manual-install.md

```diff
@@ -0,0 +1,210 @@
+---
+    description: In this guide, we'll walk through the process of deploying Ambassador Edge Stack in Kubernetes for ingress routing.
+---
+# Quick Start Installation Guide
+
+In this guide, we'll walk you through installing and configuring the Ambassador Edge Stack in your Kubernetes cluster. Within a few minutes, your cluster will be routing HTTPS requests from the Internet to a backend service. You'll also have a sense of how the Ambassador Edge Stack is managed.
+
```

Comment:
Can we explain when you would use this process instead of `edgectl install`?

Reply:
fixed